fix: update section headings to align with v1 documentation (#1343)
auth0/docs-v2: main/docs/api/authentication/device-authorization-flow/authorize-device.mdx

  The identifier of the target API (resource server) you want to access. Must match an API Identifier registered in your Auth0 tenant. Used as an alternative to `audience` when the tenant's [Resource Parameter Compatibility Profile](https://auth0.com/docs/get-started/tenant-settings#settings-advanced) is set to `compatibility`.
</ParamField>

## Response Schema

<ResponseSchema>
<ResponseField name="device_code" type="string">

>   The unique code for the device. When the user goes to the `verification_uri` in their browser-based device, this code will be bound to their session.

Reviewer: ```suggestion
  The unique code for the device. When the user goes to the `verification_uri` in their browser-based device, this code is bound to their session.
```
